### PR TITLE
chore: add js func hosted notification template

### PR DESCRIPTION
**Pull request by a1exwang**
- add js template based on ts version
- remove useless packages, scripts, config files for js template
- updated function.json
- align ignore files with function template

tested function hosted bot
image

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/13791043/

### CODE REVIEW

**Comment by a1exwang** [comment]
E2E TEST: n/a

File: templates/bot/js/notification-function-base/src/internal/messageHandler.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// the same as .initialize
```
Comment (swatDong) [inline templates/bot/js/notification-function-base/src/internal/messageHandler.js:1]: What does this mean?

Comment (a1exwang) [reply templates/bot/js/notification-function-base/src/internal/messageHandler.js]: removed

**Review by swatDong — DISMISSED** [review]

**Review by swatDong — APPROVED** [review]

File: templates/bot/js/notification-function-base/src/internal/initialize.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const { BotFrameworkAdapter } = require("botbuilder");
+const { ConversationBot } = require("@microsoft/teamsfx");
+
+// See https://aka.ms/about-bot-adapter to learn more about adapters.
+const adapter = new BotFrameworkAdapter({
+  appId: process.env.BOT_ID,
+  appPassword: process.env.BOT_PASSWORD,
```
Comment (hund030) [inline templates/bot/js/notification-function-base/src/internal/initialize.js:7]: Just a question, how are these environment variables set when local debugging?

**Review by hund030 — APPROVED** [review]

File: templates/bot/js/notification-function-base/src/internal/initialize.js (hunk 2 above)
Comment (a1exwang) [reply templates/bot/js/notification-function-base/src/internal/initialize.js:7]: 1. local debug will generate a .env.teamsfx.local after registering bot.
image
2. `npm run dev:teamsfx` will use this file.

image

**Review by Siglud — APPROVED** [review]

**Pull request merged**